### PR TITLE
fix #35 include failure reason in response

### LINKED ISSUE #35: Include failure reason
The failure reason should be included as follows `FAIL:failure reason here`.

https://github.com/activeprospect/leadconduit-integration-lowermybills/blob/b9b348939d171b473ebc3508e86fc7b0a00f9f23/src/inbound.coffee#L188

### PR DESCRIPTION
**Pull request by jgrayson676**

### CODE REVIEW

File: src/inbound.coffee [hunk 1]
```diff
@@ -185,7 +185,7 @@ response = (req, vars) ->
     if vars.outcome == 'success'
       'SUCCESS'
     else
-      'FAIL'
+      'FAIL: ' + vars.reason
```
Comment (cgrayson) [inline src/inbound.coffee]: It seems a little funky to me, too, but the format that LMB has agreed to has no spaces before or after the colon.

**Comment by cgrayson** [comment]
After that minor fix, LGTM

File: src/inbound.coffee (hunk 1 above)
Comment (ros3bud) [inline src/inbound.coffee]: What about using interpolation here?

Comment (alexkwolfe) [inline src/inbound.coffee]: Also, what happens if `vars.reason` is null or undefined?

**Pull request merged**